### PR TITLE
ci: enable `@angular-eslint/component-max-inline-declarations: error`

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/pdf-viewer/examples/2/pdf-content.component.ts [hunk 1]
```diff
@@ -6,13 +6,7 @@ import {map} from 'rxjs/operators';
...
     selector: 'tui-pdf-content',
-    template: `
-        <iframe
-            *ngIf="src$ | async as src; else loading"
-            [src]="src"
-        ></iframe>
-        <ng-template #loading><tui-loader size="xl"></tui-loader></ng-template>
-    `,
+    templateUrl: './pdf-content.component.html',
     styles: [
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/pdf-viewer/examples/2/pdf-content.component.ts:10]: GitHub Actions / Format:

> `styles` has too many lines (7). Maximum allowed is 3 (https://angular.io/guide/styleguide#style-05-04)

File: projects/demo/src/modules/components/pdf-viewer/examples/2/pdf-content.component.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<iframe
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/pdf-viewer/examples/2/pdf-content.component.html:1]: GitHub Actions / Format:

> Missing `title` attribute in <iframe>

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/pdf-viewer/examples/2/pdf-content.component.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/pdf-viewer/examples/2/pdf-content.component.ts]: fixed

File: projects/demo/src/modules/components/pdf-viewer/examples/2/pdf-content.component.html (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/pdf-viewer/examples/2/pdf-content.component.html:1]: fixed

**Pull request merged**